core/jobs_test.py

```diff
@@ -893,12 +893,12 @@ def test_events_coming_in_while_batch_job_is_running(self):
                 StartExplorationEventCounter.get_count(self.EXP_ID), 0)
 
             # Enqueue the batch computation. (It is running on 0 events).
-            StartExplorationEventCounter._kickoff_batch_job()  # pylint: disable=protected-access
+
             # Record an event while this job is in the queue. Simulate
             # this by directly calling on_incoming_event(), because using
             # StartExplorationEventHandler.record() would just put the event
             # in the task queue, which we don't want to flush yet.
-            event_services.StartExplorationEventHandler._handle_event(  # pylint: disable=protected-access
+            stats_models.StartExplorationEventLogEntryModel.create(
```

Comment:
Please update the comments as well, they are both currently incorrect.

Reply:
Sure, Done @brianrodri. Please guide me if I want to add any other information in the comments.
Thanks